src/structures/Guild.js

```diff
@@ -305,6 +305,26 @@ class Guild extends Base {
      */
     if (typeof data.max_presences !== 'undefined') this.maximumPresences = data.max_presences || 25000;
 
+    /**
+     * The approximate amount of members the guild has
+     * <info>You will need to fetch the guild using {@link Guild#fetch} if you want to receive this parameter</info>
+     * @type {?number}
+     * @name Guild#approximateMemberCount
+     */
+    if (typeof data.approximate_member_count !== 'undefined') {
+      this.approximateMemberCount = data.approximate_member_count || null;
```

Comment:
wouldn't this be undefined if the guild wasnt fetched? however the typings say that it should be null
or is the approximate_member_count field sent as null in the initial GUILD_CREATE?